internal/compose/convert.go

```diff
@@ -79,18 +101,17 @@ func ConvertSpec(spec *score.Workload) (*compose.Project, ExternalVariables, err
 			Volumes:     volumes,
 		}
 
-		var proj = compose.Project{
-			Services: compose.Services{
-				svc,
-			},
+		if len(spec.Containers) > 1 {
+			// if we have more than 1 container then namespace them with the container name
+			svc.Name += "-" + containerName
```

Comment:
This structure can cause accidental name clashes, e.g. one workload calls `workload-main` and another called `workload` with 2 containers called `main` and `other`. Both would generate the same service name.

Maybe always append container name (not just for workloads with more than one container)? This also has the advantage of not changing the structure of the file if a container is subsequently added.

Reply:
Hmm yeah I guess this would be necessary. I was avoiding it so that it didn't change existing folks workflows if they relied on a particular container name. Will change..

Reply:
Done in fba396cbf5481f4362baa476d9ff0a76e22f3236